unit-test/helpers/tds.es6.js

```diff
@@ -3,23 +3,22 @@
  * stub out the get and fallback functions
  */
 const tdsStorage = require('../../shared/js/background/storage/tds.es6')
-const tdsData = {
-    tds: require('./../data/tds.json'),
-    surrogates: require('./../data/surrogates.js').surrogates,
-    ClickToLoadConfig: require('./../data/clickToLoadConfig.json'),
-    config: require('./../data/extension-config.json')
-}
 
-tdsData.config.features.trackerAllowlist = {
```

Comment:
this was the hardcoded config that I moved to the test file from the stub and, instead, stub now allows for passing in custom config